src/main/java/org/isf/generaldata/GeneralData.java

```diff
@@ -139,6 +140,7 @@ public final class GeneralData extends ConfigurationProperties {
 	private static final String DEFAULT_PATIENTBILLSTATEMENT = "PatientBillStatement";
 	private static final boolean DEFAULT_DEBUG = false;
 	private static final int DEFAULT_STRONGLENGTH = 10;
+	private static final int DEFAULT_SESSIONTIMEOUT=300000;
```

Comment:
To make the user experience an little easier can this be in minutes instead of seconds; the code can do the math better than the user most likely. :smirk: